install-dev/data/db_structure.sql

```diff
@@ -1647,6 +1647,7 @@ CREATE TABLE `PREFIX_product_attribute_shop` (
   `low_stock_alert` TINYINT(1) NOT NULL DEFAULT 0,
   `available_date` date DEFAULT NULL,
   PRIMARY KEY (`id_product_attribute`, `id_shop`),
+  INDEX (`default_on`),
```

Comment:
I don't see why you need to define an index to fix the bug: is it for a performance improvement, and if it is can you explain me why?

Reply:
This is a mistake, indeed. It is supposed to be in https://github.com/PrestaShop/PrestaShop/pull/8534 only

Reply:
Oh sorry, I did something wrong when making PRs i guess. I actually don't know why this was sent here. Maybe some problem with github desktop.